src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -601,11 +604,29 @@ UCS_CLASS_INIT_FUNC(uct_rdmacm_cm_t, uct_component_h component,
         goto err_destroy_ev_ch;
     }
 
+    if (rdmacm_config->src_addr[0] == '\0') {
+        self->config.src_addr = NULL;
+    } else {
+        self->config.src_addr = ucs_calloc(1, sizeof(struct sockaddr),
```

Comment:
shouldn't this be sizeof sockaddr_storage?

Reply:
fixed